docs/en/Deployment.md

```diff
@@ -0,0 +1,159 @@
+
+- Advantages: 
+  1. Even if the disk is corrupted, very few messages will be lost and the real-time performance of the message will not be affected.
+  2. At the same time, when Master is down, consumers can still consume messages from Slave, and the process is transparent to the application itself and does not require human intervention.
+  3. Performance is almost as high as multiple Master mode.
+- Disadvantages:
+  1. A small number of messages will be lost when Master is down and the disk is corrupted.
+
+The starting steps for multiple Master and multiple Slave mode are as follows:
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+**2）Start the Broker cluster**
+
+```shell
+### For example, starting the first Master on machine A, assuming that the configured NameServer IP is: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-a.properties &
+ 
+### Then starting the second Master on machine B, assuming that the configured NameServer IP is: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-b.properties &
+ 
+### Then starting the first Slave on machine C, assuming that the configured NameServer IP is: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-a-s.properties &
+ 
+### Last starting the second Slave on machine D, assuming that the configured NameServer IP is: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-b-s.properties &
+```
+
+The above shows a startup command for 2M-2S-Async mode, similar to other nM-nS-Async modes.
+
+### 4 Multiple Master And Multiple Slave Mode-Synchronous dual write
+
+In this mode, one Slave is also configured for each Master and there are multiple pairs of Master-Slave.HA uses synchronous double-write, that is, the success response will be returned to the application only when the message is successfully written into the master node and replicated to more than one slave node.
+
+The advantages and disadvantages of this model are as follows:
+
+- Advantages: 
+  1. Neither the data nor the service has a single point of failure. 
+  2. In the case of Master downtime, the message is also undelayed. 
+  3. Service availability and data availability are very high;
+- Disadvantages:
+  1. The performance in this mode is slightly lower than in asynchronous replication mode (about 10% lower).
+  2. The RT sending a single message is slightly higher, and the current version cannot automatically switch to the Master after the primary node is down.
```

Comment:
"and under the current version, the slave node cannot automatically switch to the Master after the master node is down"